tests/contract-playground/src/U4.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "./U5.sol";
+
```

Comment:
This is not caught, because although it's not used in this file, it is used in UnusedImports.sol . So basically U4.sol serves as a vessel to server contents from U5.sol to UnusedImports.sol